Cargo.toml

```
@@ -148,7 +148,7 @@ moka = "0.12"
 nalgebra = "0.33"
 notify = "8.0"
 num_cpus = "1.16"
-object_store_opendal = "0.49.0"
+object_store_opendal = "0.50.0"
```

According to our code style
```suggestion
object_store_opendal = "0.50"
```